fabric-lifecycle-events-v1/src/main/java/net/fabricmc/fabric/mixin/event/lifecycle/MinecraftServerMixin.java

```diff
@@ -73,4 +89,28 @@ private void closeWorld(CallbackInfo ci, Iterator<ServerWorld> worlds, ServerWor
 			ServerBlockEntityEvents.BLOCK_ENTITY_UNLOAD.invoker().onUnload(blockEntity, serverWorld);
 		}
 	}
+
+	// The locals you have to manage for an inject are insane. And do it twice. A redirect is much cleaner.
+	// Here is what it looks like with an inject: https://gist.github.com/i509VCB/f80077cc536eb4dba62b794eba5611c1
+	@Redirect(method = "createWorlds", at = @At(value = "INVOKE", target = "Ljava/util/Map;put(Ljava/lang/Object;Ljava/lang/Object;)Ljava/lang/Object;"))
```

Comment:
Doesn't that `@At` need a `remap = false` so that the mixin AP doesn't complain?

Reply:
Probably a larger issue tbf since the mixin ap screechs on half the modules